tg-bot/src/main/java/com/tutrit/tgbot/example/bean/UserEmployee.java

```diff
@@ -0,0 +1,105 @@
+package com.tutrit.tgbot.example.bean;
+
+import java.util.Objects;
+
+public class UserEmployee {
```

Comment:
We did'n wanted userEmpoyee class according to issue you should create Engineer bean